test/corpus/literals.txt

```diff
@@ -21,41 +45,101 @@ strings
 
 "string"
 "string\""
+"string\\"
+"\115\116\114"
+"\x32\u{33}"
 
 ---
 
 (program
+  (string)
+  (string (escape_sequence))
+  (string (escape_sequence))
+  (string (escape_sequence) (escape_sequence) (escape_sequence))
+  (string (escape_sequence) (escape_sequence)))
+
+===
+colon strings
+===
+
+:test
+:*
+:#
+:?
+:-
+:+
+:9
+:_
+:/
+:<
+:>
+:=
+:^
+
+---
+
+(program
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
+  (string)
   (string)
   (string))
 
 ===
-quoted value
+quote
 ===
 
 '[]
 `123
 `(+ ,a ,b)
 `("")
+'(fn)
+(quote (each))
+'(let [x (each [y] z)])
 
 ---
 (program
-  (quoted_value 
-    (sequential_table))
-  (quoted_value
-   (number))
-  (quoted_value
-    (function_call 
-      name: (identifier)
-      (unquoted_value
-        (identifier))
-      (unquoted_value
-        (identifier))))
-  (quoted_value
-    (function_call
-      name: (string))))
+  (quote
+    (quoted_sequential_table))
```

Comment:
What was the thinking behind making `quoted_<value_type>` within the the `quote`.

Reply:
So that `'[{:foo (fn)} {:bar (each)}]` works.  The `(fn)` and the `(each)` should not be parsed as their respective rules.